zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -659,7 +660,14 @@ def build_default_context():
         'jsonpath_parse': jsonpath_rw.parse,
         'jsonpath_flat_filter': jsonpath_flat_filter,
         'filter_metrics': check_filter_metrics,
-        'filter_metric': check_filter_metric
+        'filter_metric': check_filter_metric,
+        # Some Mathematical functions from the math package:
+        'exp': math.exp,
+        'log': math.log,
```

Comment:
Was just thinking that ``log`` could be a little confusing, since one would think it is related to logging (which is a feature we might consider in the future to facilitate check/alert debugging).

I think we better off go with ``'math': math`` and let users decide which methods to use. 

what do you think?